src/app/api/sentry-example-api/route.ts

```
@@ -0,0 +1,9 @@
+import { NextResponse } from 'next/server'
+
+export const dynamic = 'force-dynamic'
+
+// A faulty API route to test Sentry's error monitoring
+export function GET() {
+  throw new Error('Sentry Example API Route Error')
+  return NextResponse.json({ data: 'Testing Sentry Error...' })
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)